platform_disk_api/api.py

```diff
@@ -123,22 +123,27 @@ def _disk_cluster_uri(self) -> str:
     def _get_org_disks_uri(self, org_name: str) -> str:
         return f"{self._disk_cluster_uri}/{org_name}"
 
-    def _get_user_disk_uri(self, user: User, org_name: Optional[str]) -> str:
+    def _get_user_disk_uri(
+        self, user: User, org_name: Optional[str], project_name: Optional[str]
+    ) -> str:
         if org_name:
-            return f"{self._get_org_disks_uri(org_name)}/{user.name}"
-        return f"{self._disk_cluster_uri}/{user.name}"
+            return f"{self._get_org_disks_uri(org_name)}/{project_name}/{user.name}"
+        return f"{self._disk_cluster_uri}/{project_name}/{user.name}"
```

Comment:
I've been thinking we might want to either use `project` or `owner`, but not the two together nested.  Since now the project is a unit of sharing.

Additionally, it might be complicated to produce a URI by a user if they do not know who the owner of the resource is.

Reply:
Also, it seems, If we add another level here, we will probably break backward-compatibility. The old URIs representing existing resources are shorter than the new ones.

Reply:
done, please take a look